src/main/java/net/minecraftforge/client/model/ModelLoader.java

```diff
@@ -391,10 +393,10 @@ public int compare(Item i1, Item i2)
             setBucketModel(Items.WATER_BUCKET);
             setBucketModel(Items.LAVA_BUCKET);
             // milk bucket only replaced if some mod adds milk
-            if(FluidRegistry.isFluidRegistered("milk"))
+            if(FluidDictionary.nameExists("milk"))
             {
                 // can the milk be put into a bucket?
-                Fluid milk = FluidRegistry.getFluid("milk");
+                Fluid milk = FluidDictionary.getFirstFluid("milk");
```

Comment:
All fluids can be put in the universal bucket unless it is disabled. 
There are some odd checks in the original code, I think the change here is valid.